lib/constants.js

```diff
@@ -36,7 +43,9 @@ export const MULTICODEC_P521_SECRET_KEY_HEADER = new Uint8Array([0x88, 0x26]);
 export const ECDSA_CURVE = {
   P256: 'P-256',
   P384: 'P-384',
-  P521: 'P-521'
+  P521: 'P-521',
+  // ensure compatibility with https://github.com/PeculiarVentures/webcrypto
+  secp256k1: 'K-256'
```

Comment:
Hmm, I'm not sure whether `'K-256'` or `'secp256k1'` is what is commonly used (as the string value), e.g., in JWKs. This is fine if it's the commonly used value, but we don't want to pick something just for compatibility with a particular library. We might need some additional translation code, I don't know. Just need a bit more research / link to RFCs, common libs (like panva's node.js jose lib) or some other docs in support of this approach. It's fine (but unfortunate) if we have to support two different expressions of this curve.

Reply:
Ok so I couldn't find a clear nomenclature definition about this.

However, section 2.1 of this file https://www.secg.org/sec2-v2.pdf does name `k` for Koblitz curve, and this paragraph has quite a sensible explanation about why it could be named `K-256`: https://lib.rs/crates/k256#readme-about-secp256k1-k-256.

But it seems the more common way is to call it `secp256k1`, but it's also not consistent with calling `secp256r1` `P-256`. 

I'll let you be the judge as this is ultimately your package, my inclination is to keep things simple with `K-256` support only (and document it) as it is a valid and consistent naming, although a bit uncommon at this point in time.

Reply:
In the NIST document about sanctioned algorithm, the nomenclature is indeed as detailed in the rust package's paragraph: https://nvlpubs.nist.gov/nistpubs/SpecialPublications/NIST.SP.800-186.pdf (P, K, B).

Appendix H.2 mentions `secp256k1` without explicitly naming it K-256.